### ISSUE TITLE
Font on EFI  boot screen too small on HiDPI monitor

### ISSUE THREAD

**Issue by TraceyC77** [getsolus/packages#1267]
### Summary

Recently, changes were made to make the font on the boot screen more legible. This has not affected the boot screen on an EFI system connected to a 1440p HiDPI ultrawide monitor. 

grub2: Improve default resolution settings getsolus/packages/pull/1063

### Steps to reproduce

1. Ensure the boot screen displays per [the help docs](https://help.getsol.us/docs/user/quick-start/boot-management)
2. Start / restart the system so you can see the boot menu

### Expected result

The font is large enough to be easily readable

### Actual result

The font is very small and is difficult to read. Compare to booting Solus in a VM which has been improved.

### Environment

- [X] Is system up to date?

### Repo

Unstable

### Desktop Environment

Plasma

### System details

Operating System: Solus 4.5
KDE Plasma Version: 5.27.10
KDE Frameworks Version: 5.112.0
Qt Version: 5.15.11
Kernel Version: 6.6.9-269.current (64-bit)
Graphics Platform: X11
Processors: 16 × 11th Gen Intel® Core™ i7-11800H @ 2.30GHz
Memory: 62.5 GiB of RAM
Graphics Processor: NVIDIA GeForce RTX 3060 Laptop GPU/PCIe/SSE2
Manufacturer: Dell Inc.
Product Name: XPS 17 9710

External Monitor: LG 34GK950F 3440 x 1440

### Other comments

The file /etc/default/grub is not present on the system.
Systemd has an issue that has supposedly resolved the issue
[Sdboot setmode v2](https://github.com/systemd/systemd/pull/8086)

**Comment by TraceyC77** [comment]
I just had a conversation on Matrix with Neal Gompa, and learned that *currently* this is only possible to set the font size at kernel compile time. He's investigating a different approach, but this is in very early stages. Just noting to set expectations for Solus. 

Given that we don't provide custom kernels, we probably need to wait for an alternative to surface. Changing from bug to feature accordingly.

**Comment by devlocalhost** [comment]
i did some testing and messing around, and i think its caused by plymouth, but keep reading.

i figured out how to change the font size: 
- `echo "nosplash loglevel=7 systemd.show_status=1 rd.vconsole.font=ter-v12n" | sudo tee /etc/kernel/cmdline.d/verbose.conf`
- `echo "plymouth.enable=0" | sudo tee /etc/kernel/cmdline.d/noplymouth.conf`
-  `sudo clr-boot-manager update`
- and finally `reboot`

but wait, `rd.vconsole.font` is already present in the defaults (default kernel cmdline), and for me its set to `ter-v32b`, which is the biggest size, and very comfortable/readable on my 16:10 2560x1600 laptop. 

so, maybe it isnt plymouth's fault? because if you check the command i sent earlier, i choose `ter-v12n`, the smallest font. after a reboot, i did see it. the font was changed and it was exactly what i picked, but just before GDM showed up, the font quickly went back to `ter-v32b`, which is the default.

and then i remembered there is `/etc/vconsole.conf`. so i also changed the font there (which, here too, was `ter-v32b` for me): `FONT=ter-v12n`, and success! the font stays, and does not change. i also experimented with `ter-v14b`. same effect. it works.

tldr: i suspect its plymouth and `vconsole.conf` (or systemd-vconsole-setup?). this command: `echo "nosplash loglevel=7 systemd.show_status=1 rd.vconsole.font=ter-v14b" | sudo tee /etc/kernel/cmdline.d/verbose.conf && sudo clr-boot-manager update` and this change: `FONT=ter-v14b` to `/etc/vconsole.conf` allowed me to change the font size.
